ibazel/ibazel.go

```diff
@@ -250,23 +268,26 @@ func (i *IBazel) setup() error {
 // Run the specified target (singular) in the IBazel loop.
 func (i *IBazel) Run(target string, args []string) error {
 	i.args = args
-	return i.loop("run", i.run, []string{target})
+	cmd := RunnableCommand{command: i.run, cancelableCommand: i.runWithEarlyAbort}
+	return i.loop("run", cmd, []string{target})
 }
 
 // Build the specified targets in the IBazel loop.
 func (i *IBazel) Build(targets ...string) error {
-	return i.loop("build", i.build, targets)
+	cmd := RunnableCommand{command: i.build, cancelableCommand: i.buildWithEarlyAbort}
```

Comment:
How does the user get to choose between early abort vs non early abort? 
I think you should make this configurable via a flag on the bazel target or a flag to ibazel itself (maybe both)
The default should remain the same so the change is not breaking

Reply:
For the time being, and at least to begin with, it can be achieved through the usage of a ENV VAR `IBAZEL_ABORT_COMPILATION_EARLY=1` 😃